public/server/db/rcmp.json

```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "Royal Canadian Mounted Police"
```

Comment:
Abbreve is sort of an 'unofficial' library, so I don't think we could accept this particular entry. I like all the others, however.